### PR TITLE
Image Caption Generator

### LINKED ISSUE #281: Image Caption Generator
### Project Information
The objective of the project is to predict the captions for the input image. The dataset consists of 8k images and 5 captions for each image. The features are extracted from both the image and the text captions for input. The features will be concatenated to predict the next word of the caption. CNN is used for image and LSTM is used for text. BLEU Score is used as a metric to evaluate the performance of the trained model.

### Libraries

- numpy
- matplotlib
- keras
- tensorflow
- nltk

### Neural Network

- VGG16 Network
- CNN-LSTM Network

### PR DESCRIPTION
**Pull request by deepanshubaghel**
### Project Information
The objective of the project is to predict the captions for the input image. The dataset consists of 8k images and 5 captions for each image. The features are extracted from both the image and the text captions for input. The features will be concatenated to predict the next word of the caption. CNN is used for image and LSTM is used for text. BLEU Score is used as a metric to evaluate the performance of the trained model.

### Libraries

- numpy
- matplotlib
- keras
- tensorflow
- nltk

### Neural Network

- VGG16 Network
- CNN-LSTM Network

fixes #281

### CODE REVIEW

**Comment by suryanshsk** [comment]
Hi, thank you for your interest in the project! Just a reminder, this
repository is focused specifically on the *Python Voice Assistant*. Please
make sure that any issues or PRs are relevant to this project's scope.
Unrelated features or suggestions should be directed elsewhere. Thanks for
understanding!

**Comment by suryanshsk** [comment]
Hi, thank you for your interest in the project! Just a reminder, this repository is focused specifically on the Python Voice Assistant. Please make sure that any issues or PRs are relevant to this project's scope. Unrelated features or suggestions should be directed elsewhere. Thanks for understanding!
